### DISCUSSION TITLE
Git and ignored files

### DISCUSSION THREAD

**Discussion by LmG7119** [dbeaver/dbeaver#35756] [Q&A]
What files and folders should be part of the ignore? 
I added the following 
/.project
/.project-metadata.json.bak
/project-metadata.json

How do I restore those if I need to inside the DBeaver?

**Comment by TB-HunBoka** [comment]
I think it is better practice to only commit files you actually worked on.
I use the gitignore for sensitive information like: passwords, apikeys and tmp folders.
